nearlib/test/integration.test.js

```diff
@@ -21,6 +25,69 @@ test('test creating default config', async () => {
     Near.createDefaultConfig();
 });
 
+describe("dev connect", () => {
+    test('test dev connect with no account creates a new account', async () => {
+        const keyStoreNoAccount = new InMemoryKeyStore();
+        const localStorage = createMockLocalStorage();
+        const nearNoAccount = await dev.connect({
+            nodeUrl: 'http://localhost:3030',
+            deps: {
+                keyStore: keyStoreNoAccount,
+                localStorage: localStorage,
+                createAccount: (async (newAccountName, newAccountPublicKey) => {
```

Comment:
shouldn't this version of `createAccount` be available outside of tests as well?

Reply:
I'll move it when there's need. Not sure where to put it now